pytometry/tools/_humap.py

```diff
@@ -0,0 +1,439 @@
+from scipy.stats import entropy
+from scipy.special import softmax
+# import multiprocessing as mp
+import numpy as np
+from scipy.sparse import csr_matrix, vstack, hstack
+from scanpy.tools._umap import umap as embed_umap
+from anndata import AnnData
+# import matplotlib.pyplot as plt
+from matplotlib.widgets import LassoSelector
+from matplotlib.path import Path
+
+HELPER_VAR = dict()
+
+class _Scale:
+    """
+    Class that contains information about a Scale
+    Properties:
+    self.X
+       datapoints
+    self.T
+       Transition matrix
+    self.I
+       Area of Influence matrix
+    self.W
+       Weight vector
+    self.P
+       Probability distribution
+    self.X_hsne
+       Embedding positions
+    self.lm_ind
+       landmark indices
+    self.parent_scale
+       parent scale (_Scale object)
+
+
+    """
+    def __init__(self, X=None, T=None, I=None, W=None, P=None, X_hsne=None, lm_ind=None, parent_scale=None):
```

Comment:
Argument `I` is ambiguous. Please change the name. Replace argument `X_hsne` by `X_humap` for the `_humap.py` function.